src/services/resource.js

```diff
@@ -1560,6 +1560,76 @@ module.exports = class resourceHelper {
 				delete filterQuery.is_reusable
 			}
 
+			let getOrgPolicies = await organizationConfigQueries.findAll(
+				{
+					organization_code: organization_code,
+					tenant_code: tenant_code,
+				},
+				['organization_code', 'external_project_resource_visibility_policy']
+			)
+			if (getOrgPolicies.length > 0) {
```

Comment:
@praveenKDass Can we write a generic function to filterResourcesBasedOnSaasPolicy

this will return the filterQuery